tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/configurables/ConfigurableTest.java

```diff
@@ -87,7 +89,50 @@ public void testAPICNegativeTest() throws BallerinaTestException {
     @Test
     public void testEnvironmentVariableBasedConfigFile() throws BallerinaTestException {
         String configFilePath = Paths.get(testFileLocation, "config_files", "Config.toml").toString();
-        executeBalCommand("", testsPassed, "run", "envVarPkg", addEnvVariables(configFilePath));
+        executeBalCommand("", testsPassed, "run", "envVarPkg", addConfigPathVariable(configFilePath));
+    }
+
+    @Test
```

Comment:
I think most os these test can be moved to runtime unit tests.

Reply:
I'll be moving these tests with this issue #29258